### ISSUE TITLE
Modificated AlexNet by Zeiler/Fergus

### ISSUE THREAD

**Issue by l-ew** [eladhoffer/ImageNet-Training#10]
Hello,

this is not related to your code, but I still hope you can help. I want to replicate the modified AlexNet by Zeiler and Fergus of 2013 ("Visualizing and Understanding Convolutional Networks") but they spare some details.

1) What is their exact learning rate schedule? They just write "We anneal the learning rate throughout training manually when the validation error plateaus".

2) Do they use weight decay?

3) In which layers do they "renormalize" the filters (they do not divide the input by the global standard deviation)?

Would be glad if you could answer any of these or if you have any further reference.
